ccp/lib/utils.py

```
@@ -0,0 +1,83 @@
+"""
+This file contains are the basic utility functions to be used by the clients.
+"""
+import json
+import base64
+import subprocess
+import requests
+
+
+def request_url(
+        request, params=None, verify_ssl=False, auth=None, headers=None
+):
+    """
+    Queries a specified url and returns data, if any
+    :param request: The url to send the request to.
+    :type request str
+    :param params: Any params that are to be passed on to the request
+    :param verify_ssl: IF true, ssl errors are ignored
```

`s/IF/If`

and `If False, ssl errors are ignored` ?

---

Sorted